src/features/amUI/landingPage/LandingPage.tsx

```diff
@@ -0,0 +1,237 @@
+import {
+  getConsentMenuItem,
+  getPoaOverviewMenuItem,
+  getReporteesMenuItem,
+  getRequestsMenuItem,
+  getSettingsMenuItem,
+  getSystemUserMenuItem,
+  getUsersMenuItem,
+} from '@/resources/utils/sidebarConfig';
+
+export const LandingPage = () => {
+  const { t } = useTranslation();
+  const { data: reportee } = useGetReporteeQuery();
+  const { data: isAdmin } = useGetIsAdminQuery();
+  const { data: isClientAdmin } = useGetIsClientAdminQuery();
+  const { data: canAccessSettings } = useGetIsCompanyProfileAdminQuery();
+
+  const reporteeName = formatDisplayName({
+    fullName: reportee?.name || '',
+    type: reportee?.type === 'Organization' ? 'company' : 'person',
+  });
+
+  const getMenuItems = (): MenuItemProps[] => {
+    const displayConfettiPackage = window.featureFlags?.displayConfettiPackage;
+    const displayConsentGui = window.featureFlags?.displayConsentGui;
+    const displayPoaOverviewPage = window.featureFlags?.displayPoaOverviewPage;
+
+    if (!reportee) {
+      const loadingMenuItem: MenuItemProps = {
+        icon: {
+          name: 'xxxxx xxxxxxxxx',
+          type: 'person',
+        },
+        title: 'xxxxxxxxxxxxxxx',
+        description: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+        href: '',
+        loading: true,
+      };
+      return [loadingMenuItem, loadingMenuItem, loadingMenuItem, loadingMenuItem];
+    }
+
+    const items: MenuItemProps[] = [];
+    if (displayConfettiPackage) {
+      items.push({
+        ...getUsersMenuItem(),
+        description:
+          reportee.type === 'Organization'
+            ? t('landing_page.users_item_description_org')
+            : t('landing_page.users_item_description_person'),
+      });
+    }
+
+    if (displayPoaOverviewPage && isAdmin) {
+      items.push({
+        ...getPoaOverviewMenuItem(),
+        description: t('landing_page.poa_item_description'),
+      });
+    }
+
+    if (displayConfettiPackage && isAdmin) {
+      items.push({
+        ...getReporteesMenuItem(),
+        description:
+          reportee.type === 'Organization'
+            ? t('landing_page.reportees_item_description_org')
+            : t('landing_page.reportees_item_description_person'),
+      });
+    }
+
+    if (displayConsentGui && hasConsentPermission(reportee, isAdmin)) {
+      items.push({
+        ...getConsentMenuItem(),
+        description: t('landing_page.consent_item_description'),
+      });
+    }
+
+    if (
+      hasCreateSystemUserPermission(reportee) ||
+      hasSystemUserClientAdminPermission(reportee, isClientAdmin)
+    ) {
+      items.push({
+        ...getSystemUserMenuItem(),
+        description: t('landing_page.systemuser_item_description'),
+      });
+    }
+    return items;
+  };
+
+  const getRequestCountText = (
+    reportee: ReporteeInfo | undefined,
+    requestCount: number,
+  ): string => {
+    const name = reportee?.type === 'Organization' ? reporteeName : t('common.you_uppercase');
```

Comment:
Should this be a check on whether you are representing yourself, considering you can represent people as well? 🤔